docs/Draft-Generation/using-paragraph-breaks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Using Paragraph Breaks
+sidebar_position: 2
+slug: /Using-Paragraph-Breaks
+---
+# Using Paragraph Breaks
+
+The Paragraph Breaks feature provides additional options for handling the paragraph breaks in generated drafts. Paragraph breaks are brought over from the drafting source and there are three options for how to handle them. The options are named "Best Guess", "Remove", and "Move to End". 
+
+"Remove" will remove all the Paragraph breaks that appear within a verse. Paragraph breaks that are between verses are not affected.  If you use this option you will need to use Paratext to add in any paragraph break markers that are required in your translation. The "Move to End" option will move all Paragraph breaks to the end of the drafted verse text.  This was the way that Scripture Forge originally handled Paragraph breaks, before this feature was released. Scripture Forge can attempt to place paragraph markers correctly in the drafted verse and this is the default option and called "Best Guess".  Where the system works well, it should save significant work. The "Best Guess" option is especially useful for books like Psalms where there are many paragraph breaks within verses. 
+
+The button for the formatting options is on the “Generate Draft” page.  When you change the settings a preview of the changes will be shown. Changing the option does not affect drafts that have been added to a project, changing the options only affects the draft previews before they are added to a project.
+
+![](./paragraph_breaks_image1.png)
+
+Use the "Formatting Options" button to set up the paragraph breaks and quote styles for your draft. After the initial set up, you can return to the “Formatting Options” page and change your settings at any time.
+
+![](./paragraph_breaks_image2.png)
+
+# Using the Paragraph Breaks feature (step-by-step)
+
+1. Navigate to the Generate Draft page in Scripture Forge.
+2. Select the book you want to draft (e.g. 1 Kings).

```

Comment:
This isn't the right instruction; it should tell the user to click the formatting options button:

<img width="627" height="238" alt="Image" src="https://github.com/user-attachments/assets/38cf892f-5ce6-4720-a574-00c7d05db455" />